torchtune/modules/attention.py

```diff
@@ -14,9 +14,9 @@
 logger = logging.getLogger(__name__)
 
 
-class CausalSelfAttention(nn.Module):
-    """Multi-headed grouped query self-attention (GQA) layer introduced
-    in https://arxiv.org/abs/2305.13245v1.
+class MultiHeadedAttention(nn.Module):
```

Comment:
nit: isn't the canonical name `MultiHeadAttention`?

Reply:
You're right, I'll update this. 😢